Task_2/src/components/Products/ProductPage.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import ProductModel from "../../Models/ProductModel";
+import { AiOutlinePercentage } from "react-icons/ai";
+import { BsCurrencyDollar } from "react-icons/bs";
+import { GoStarFill } from "react-icons/go";
+import { FaRegHeart } from "react-icons/fa";
+import { RiShoppingCartLine } from "react-icons/ri";
+import { IoMdShare } from "react-icons/io";
+import ProductGridImage from "./ProductImageGrig";
+import { useParams } from "react-router-dom";
+
+const ProductPage: React.FC = () => {
+  const Prodcut: ProductModel = {
```

Comment:
I don't think it's needed @codal-uchoudhary

Reply:
Updated